aiowebostv/webos_client.py

```diff
@@ -617,7 +632,11 @@ async def request(self, uri, payload=None, cmd_type="request", uid=None):
         if payload is None:
             raise WebOsTvCommandError(f"Invalid request response {response}")
 
-        return_value = payload.get("returnValue") or payload.get("subscribed")
+        return_value = (
+            payload.get("returnValue")
+            or payload.get("subscribed")
+            or payload.get("subscription")
```

Comment:
Can you explain this change? Does newer devices return `subscription` instead of `subscribed`? I want to validate this since this has impact on all of the requests to the device so I wonder why the device will answer with `subscription` in this case.

Reply:
Only com.webos.media/getForegroundAppInfo does.  When you subscribe it returns:
`{'subscription': True}`

The other endpoints still use subscribed.